addons/escoria-core/game/esc_inputs_manager.gd

```diff
@@ -346,6 +348,18 @@ func _on_mouse_exited_item(item: ESCItem) -> void:
 # - item: The Escoria item clicked
 # - event: The input event from the click
 func _on_mouse_left_clicked_item(item: ESCItem, event: InputEvent) -> void:
+	if item as ESCPlayer and not (item as ESCPlayer).selectable:
```

Comment:
Shouldn't this block happen after test `if input_mode == INPUT_ALL:` that happens below?

Reply:
Yeah, that would probably save some checks. Can't remember why I put the block there but will change.